test/test.js

```diff
@@ -298,6 +309,32 @@ describe('Layer', () => {
     );
   });
 
+  it('can publish a FeatureType from PostGIS', async () => {
```

Comment:
This contains no test with e.g. expect. So no real test or is it hoping for no exception? Maybe adding some basic expectations would be a good thing

Reply:
fixed with e5f562c